javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+var width = 420;
+var height = 70;
+var area = width * height;
+
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+var bill = 29400;
+var payment = 1073;
+var change = bill - payment;
```

Comment:
Shouldn't you pay enough to cover the bill? 😂 . It should be `payment - bill` and payment should be enough to cover the bill

Reply:
my bad im dyslexic